### PR TITLE
Api Properties Builder addition 

### PR DESCRIPTION
**Pull request by Courtneyp123**

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/service/CollectorServiceImpl.java [hunk 1]
```diff
@@ -284,6 +288,20 @@ public Collector createCollector(Collector collector) {
         return (List<CollectorItem>) collectorItemRepository.findAll(ids);
     }
 
+    @Override
+    public void deletePropertiesInCollectorById(String id) {
+        ObjectId objectId = new ObjectId(id);
+        List<Collector> collectors = collectorsById(objectId);
```
Comment (nireeshT) [inline src/main/java/com/capitalone/dashboard/service/CollectorServiceImpl.java:294]: collectorsById should return a single instance of collector and not list at is findOne and not findAll.

Comment (nameisaravind) [reply src/main/java/com/capitalone/dashboard/service/CollectorServiceImpl.java:294]: collectorsById -> findById -> returns list (existing CollectorRepository class method)

Comment (Courtneyp123) [reply src/main/java/com/capitalone/dashboard/service/CollectorServiceImpl.java:294]: @nireeshT changed in core and api

Comment (Courtneyp123) [reply src/main/java/com/capitalone/dashboard/service/CollectorServiceImpl.java]: @nireeshT used findOne instead, but in the future, we may want to change findById to return overall one Collector vs. list.

**Pull request merged**